packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -967,15 +989,19 @@ export class GarbageCollector implements IGarbageCollector {
 		}
 
 		const errorRequest: IRequest = request ?? { url: nodePath };
-		// If the object is tombstoned and tombstone enforcement is configured, throw an error.
-		if (isTombstoned && this.throwOnTombstoneLoad && headerData?.allowTombstone !== true) {
-			// The requested data store is removed by gc. Create a 404 gc response exception.
-			throw responseToException(
-				createResponseError(404, `${nodeType} was tombstoned`, errorRequest, {
-					[TombstoneResponseHeaderKey]: true,
-				}),
-				errorRequest,
-			);
+		if (isTombstoned) {
```

Comment:
For (1), I searched the code for `GC_Tombstone`. The only spot outside of the Telemetry Tracker is for `GC_Tombstone_DataStore_Changed`, which is logged in `dataStoreContext.ts`.  We have a task open to move that logic here too.  We don't need to trigger autorecovery there because the Summarizer will certainly load any object that receives an op, and will trigger auto-recovery at that point.

For (2), I moved the call to `triggerAutorecovery` (renamed from `submitTombstoneLoadedOp`) to up right next to where the log happens.  It's good to trigger this for DDSes as well - there are single digits of this log across months of production traffic, so no scale concern there!